get.go

```diff
@@ -652,6 +663,12 @@ func getPackage(ctx context.Context, logger *log.Logger, c installPackageConfig,
 		return err
 	}
 
+	// We have close it once here so the defer func does not need to close it again.
+	closed = true
+	if err := tmpModFile.Close(); err != nil {
```

Comment:
Same here, it's not acceptable to use `tmpModFile` var that is closed. It might work by accident now, but will cause surprises later on.

Why not closing right before `os.Rename` (and saving tmpModFile fileapath to variable?)